### PR TITLE
Fixing mismatched result for 'Array.prototype.sort' subtest

### PR DESCRIPTION
**Pull request by afmenez**
The current results don't make sense.
- I don't have a Mac, so I haven't actually tested;
- Shouldn't all the safaritp/webkit identical results be removed when they cascade from safari/safaritp?

### CODE REVIEW

File: data-es5.js [hunk 1]
```diff
@@ -740,9 +740,8 @@ exports.tests = [
       ie9: true,
       firefox2: false,
       firefox5: true,
+      safari1: false,
       safari10_1: true,
-      safaritp: false,
```
Comment (ljharb) [inline data-es5.js:744]: Def leave these; webkit, safari, and safaritp don't necessarily have the same features.

**Review by ljharb — CHANGES_REQUESTED** [review]

File: data-es5.js (hunk 1 above)
Comment (afmenez) [reply data-es5.js:744]: Well, according to environments.json, they all belong in the same "family", so the results cascade.  But I am adding them back.

**Review by ljharb — APPROVED** [review]

File: data-es5.js (hunk 1 above)
Comment (ljharb) [reply data-es5.js:744]: Right but anything explicitly tested should never be removed - cascading is a sane default, but explicit results are never redundant.

Comment (afmenez) [reply data-es5.js:744]: Well, as I stated in the first comment I haven't explicitly tested. :D

**Pull request merged**